browser_history/cli.py

```diff
@@ -113,11 +114,12 @@ def main():
 
     try:
         if args.output is None:
+            if args.format == "infer":
+                args.format = "csv"
```

Comment:
Is this needed here? The save function already does this.

Reply:
Yeah, but that feeds into the formatted function, this is used only when you don't provide the -o argument. Should I implement it in formatted?

Reply:
Ah okay. Makes sense